cli/README.md

```diff
@@ -0,0 +1,335 @@
+oclif-hello-world
+=================
+
+oclif example Hello World CLI
+
+[![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
+[![CircleCI](https://circleci.com/gh/oclif/hello-world/tree/main.svg?style=shield)](https://circleci.com/gh/oclif/hello-world/tree/main)
+[![GitHub license](https://img.shields.io/github/license/oclif/hello-world)](https://github.com/oclif/hello-world/blob/main/LICENSE)
+
+<!-- toc -->
+* [Usage](#usage)
+* [Commands](#commands)
+<!-- tocstop -->
+# Usage
+<!-- usage -->
+```sh-session
+$ npm install -g mevflood
+$ mevflood COMMAND
+running command...
+$ mevflood (--version)
+mevflood/0.0.0 darwin-x64 node-v16.16.0
+$ mevflood --help [COMMAND]
+USAGE
+  $ mevflood COMMAND
+...
+```
+<!-- usagestop -->
+# Commands
+<!-- commands -->
+* [`mevflood hello PERSON`](#mevflood-hello-person)
+* [`mevflood hello world`](#mevflood-hello-world)
+* [`mevflood help [COMMANDS]`](#mevflood-help-commands)
+* [`mevflood plugins`](#mevflood-plugins)
+* [`mevflood plugins:install PLUGIN...`](#mevflood-pluginsinstall-plugin)
+* [`mevflood plugins:inspect PLUGIN...`](#mevflood-pluginsinspect-plugin)
+* [`mevflood plugins:install PLUGIN...`](#mevflood-pluginsinstall-plugin-1)
+* [`mevflood plugins:link PLUGIN`](#mevflood-pluginslink-plugin)
+* [`mevflood plugins:uninstall PLUGIN...`](#mevflood-pluginsuninstall-plugin)
+* [`mevflood plugins:uninstall PLUGIN...`](#mevflood-pluginsuninstall-plugin-1)
+* [`mevflood plugins:uninstall PLUGIN...`](#mevflood-pluginsuninstall-plugin-2)
+* [`mevflood plugins update`](#mevflood-plugins-update)
+
+## `mevflood hello PERSON`
+
+Say hello
+
+```
+USAGE
+  $ mevflood hello PERSON -f <value>
+
+ARGUMENTS
+  PERSON  Person to say hello to
+
+FLAGS
+  -f, --from=<value>  (required) Who is saying hello
+
+DESCRIPTION
+  Say hello
+
+EXAMPLES
+  $ oex hello friend --from oclif
+  hello friend from oclif! (./src/commands/hello/index.ts)
+```
+
+_See code: [dist/commands/hello/index.ts](https://github.com/flashbots/mev-flood/blob/v0.0.0/dist/commands/hello/index.ts)_
+
+## `mevflood hello world`
+
+Say hello world
+
+```
+USAGE
+  $ mevflood hello world
+
+DESCRIPTION
+  Say hello world
+
+EXAMPLES
+  $ mevflood hello world
+  hello world! (./src/commands/hello/world.ts)
+```
+
+## `mevflood help [COMMANDS]`
+
+Display help for mevflood.
+
+```
+USAGE
+  $ mevflood help [COMMANDS] [-n]
+
+ARGUMENTS
+  COMMANDS  Command to show help for.
+
+FLAGS
+  -n, --nested-commands  Include all nested commands in the output.
+
+DESCRIPTION
+  Display help for mevflood.
+```
+
+_See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v5.2.9/src/commands/help.ts)_
+
+## `mevflood plugins`
```

Comment:
Is mevflood becoming it's own operating system now?

Does it really need a plugin system?

Reply:
Not necessarily opposed, but it's usually best to keep things simple, unless actually needed :)

Reply:
that's just something that oclif auto-generated, we don't use it